### DISCUSSION TITLE
Hosting the files

### DISCUSSION THREAD

**Discussion by techdevmaestro** [Ultimate-Hosts-Blacklist/Ultimate.Hosts.Blacklist#694] [Ideas]
Hey guys, is it maybe possible to host the the files on the github page in the future?
I have a adguard server and to add a new list to block it needs a link because it's not supporting uploading files for blocking

**Comment by spirillen** [comment]
Which file are you having in mind?

**Comment by techdevmaestro** [comment]
the raw files

**Reply by JonnyTech** [reply]
You can grab with wget or curl, for example:

`https://github.com/Ultimate-Hosts-Blacklist/Ultimate.Hosts.Blacklist/raw/master/hosts/hosts0`

**Reply by spirillen** [reply]
Well the right URI would be `raw.githubusercontent.com` like `https://raw.githubusercontent.com/Ultimate-Hosts-Blacklist/Ultimate.Hosts.Blacklist/master/hosts/hosts0`

**Comment by funilrys** [comment]
What about [our mirrors](https://github.com/Ultimate-Hosts-Blacklist/Ultimate.Hosts.Blacklist?tab=readme-ov-file#raw-file-sources) 
 @techdevmaestro ?
